### PR TITLE
Add directory explanation to Wiki

### LINKED ISSUE #14: Add directories explanation to Wiki
## Why
As Go and Gin are new for us at Nimble, it would be great to have directories explanation to Wiki to describe the directories.
 
## Who Benefits?
Developers

### PR DESCRIPTION
**Pull request by gutakk**
## What happened 👀
Resolve #14
- Added content to [wiki](https://github.com/nimblehq/gin-templates/wiki/Directories)
- Update README
 
## Insight 📝
N/A
 
## Proof Of Work 📹
N/A

### CODE REVIEW

**Comment by junan** [comment]
Screen Shot 2021-03-02 at 11 55 04 AM

> Test helper functions.

@gutakk Can we change the text to `Test helper functions and files` as the directory will have files also(in fixtures)

**Comment by gutakk** [comment]
> Screen Shot 2021-03-02 at 11 55 04 AM
> 
> > Test helper functions.
> 
> @gutakk Can we change the text to `Test helper functions and files` as the directory will have files also(in fixtures)

Fixed.

**Review by junan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 Our templates offer a rich boilerplate to jump start Go Gin-based application development.
 
+Check this [wiki](https://github.com/nimblehq/gin-templates/wiki/Directories) for more information about template structure.
```
Comment (olivierobert) [inline README.md]: Minor syntax issue:

```suggestion
Check this [wiki](https://github.com/nimblehq/gin-templates/wiki/Directories) for more information about the template structure.
```

Comment (gutakk) [reply README.md]: Fixed in 9fd0761bd4fafc96779ed8c62e32903e84f2c583

**Review by olivierobert — APPROVED** [review]

**Pull request merged**